src/platform/plugins/shared/management/public/plugin.tsx

```diff
@@ -131,8 +131,17 @@ export class ManagementPlugin
               const [, ...trailingBreadcrumbs] = newBreadcrumbs;
               deps.serverless.setBreadcrumbs(trailingBreadcrumbs);
             } else {
-              coreStart.chrome.setBreadcrumbs(newBreadcrumbs, {
-                project: { value: newBreadcrumbs, absolute: true },
+              coreStart.chrome.getActiveSolutionNavId$().subscribe((activeSolutionId) => {
```

Comment:
The code in this function is getting long and contains a fair amount of duplication. The conditional logic only needs to wrap the set of breadcrumbs while `coreStart.chrome.setBreadcrumbs` only needs to be called once.

Reply:
Hey @mattkime thank you for reviewing this. I found a potential bug with this approach. I am modifying this logic a bit. Will keep posted when I have an update with a better solution.